demo/web/completion/index.html

```diff
@@ -112,6 +117,7 @@ <h1>picoLLM Web Completion Demo</h1>
         </div>
         <div style="display: flex; justify-content: space-between; margin-top: 5px;">
           <span>Tokens per second: <span id="tokensPerSec">-</span></span>
+          <button id="interrupt">Stop</button>
```

Comment:
for completion and chat demos can we have the Generate/Send button just turn into the stop button? This is how chatgpt does it, and the dangling stop button that appears looks a bit weird